core/controllers/question_editor.py

```diff
@@ -49,15 +64,22 @@ def post(self, skill_id):
         question_dict['question_state_data_schema_version'] = (
             feconf.CURRENT_STATE_SCHEMA_VERSION)
         question_dict['id'] = question_services.get_new_question_id()
+        question_dict['linked_skill_ids'] = skill_ids
         try:
             question = question_domain.Question.from_dict(question_dict)
-        except:
+        except Exception:
             raise self.InvalidInputException
         question_services.add_question(self.user_id, question)
         # TODO(vinitamurthi): Replace DEFAULT_SKILL_DIFFICULTY
         # with a value passed from the frontend.
-        question_services.create_new_question_skill_link(
-            question.id, skill_id, constants.DEFAULT_SKILL_DIFFICULTY)
+        try:
+            question_services.create_multi_question_skill_links_for_question(
+                self.user_id,
+                question.id,
+                skill_ids,
+                [constants.DEFAULT_SKILL_DIFFICULTY] * len(skill_ids))
+        except Exception:
```

Comment:
It can occur when the skill ids and skill difficulties do not match (see question_Services.py). As of now that will never happen but it can happen in the future when we pass in skill difficulties from the frontend.

Reply:
Alright I will do that when we move to using skill difficulties from the fontend, removing the try/except for now.

Reply:
Done